### ISSUE TITLE
Curriculum Review: Day1 Matplotlib & astropy

### ISSUE THREAD

**Issue by pranav-satheesh** [semaphoreP/codeastro#54]
## Prior Knowledge

- Working knowledge on NumPy array is needed but this is already covered in the previous module.
- Need to check if `astropy` and `orbitize` are already installed prior to the activities.

## Content

- In the first snippet, the `figure` object link that redirects to matplotlib documentation isn't working. Needs to be modified. Here's the working [link](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.figure.html).
-  In the second snippet, we should explain how `plt.errorbar` is different from `plt.plot`. Also, explain the markers and linestyle used.
- In the subplots section, we can fill the remaining subplots with different styles and colours to make it prettier.
- We can also add one snippet that plots both arrays in one plot and add `labels` and use `plt.legend` for displaying the legend.

## Learning Goals

The whole module covers both matplotlib and astropy pretty well. The activities are all really good especially the astropy ones. Hope we get to see some really nice plots from the teams 😄

**Comment by sblunt** [comment]
Thanks for the very helpful suggestions @pranav-satheesh! Here's how I addressed them:

- fixed the link
- added more descriptive comments in first snippet 
- I think I'm going to keep the 4-panel subplot somewhat empty, since I want to make it easy(er) to recognize which commands are acting on which panels. I did change the colors in the 2-panel subplots to make them a bit prettier though-- let me know what you think!
- added an example to the 4-panel figure of using `legend()` and plotting two different arrays on the same plot
